verilog/formatting/formatter_test.cc

```diff
@@ -887,28 +887,31 @@ static const std::initializer_list<FormatterTestCase> kFormatterTestCases = {
         "  end\n"
         "endmodule\n",
     },
-
     {
         // clocking declarations in modules
         " module mcd ; "
         "clocking   cb @( posedge clk);\t\tendclocking "
-        "clocking cb2   @ (posedge  clk\n); endclocking endmodule",
+        "clocking cb2   @ (posedge  clk\n); input a; output b; endclocking endmodule",
         "module mcd;\n"
         "  clocking cb @(posedge clk);\n"
         "  endclocking\n"
         "  clocking cb2 @(posedge clk);\n"
+        "    input a;\n"
+        "    output b;\n"
         "  endclocking\n"
         "endmodule\n",
```

Comment:
Let's keep both versions of this test, empty clocking declaration (still syntactically valid), and with the new ports you added.

Reply:
Doesn't the first one ("cb") still matches the empty clocking block case?

I noticed that the original test seemed to have two clocking blocks to handle #31 (incorrect handling of two clocking blocks). This still seemed to be covered by my adjusted version, which is why I adjusted the second block instead of adding a third case.

But I'll add a "cb3" and restore the original test if that is preferred.

Reply:
Yes, even though the larger test 'covers' multiple aspects, we would still prefer to have these tests as small as possible so that a failing test case is likely to narrow down the problem.  For example, two empty clocking declarations tests specifically for the interaction between two consecutive declarations.  If you like, the 'new' test case can just have one clocking declaration, because what you are fixing is internal to a single declaration.